extras/tmux/evergarden.tmux

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Set path of script
+PLUGIN_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+echo $PLUGIN_DIR
+
+tmux source "${PLUGIN_DIR}/evergarden_tmux.conf"
```

Comment:
since this is a bash file it should probably either:
 - not have a file extension (the filetype would be assumed based on the shebang line)
 - have the `.sh` extension

Reply:
Changed to `evergarden.sh` and `evergarden.conf`